String/CheckExceeding.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * @function checkExceeding
+ * @description - Exceeding words are words where the gap between two adjacent characters is increasing. The gap is the distance in ascii
+ * @param {string} str
+ * @returns {boolean}
+ * @example - checkExceeding('delete') => true, ascii difference - [1, 7, 7, 15, 15] which is incremental
+ * @example - checkExceeding('update') => false, ascii difference - [5, 12, 3, 19, 15] which is not incremental
+ */
+const checkExceeding = (str) => {
+  if (typeof str !== 'string') {
+    throw new TypeError('Argument is not a string')
+  }
+
+  const upperChars = str
+    .toUpperCase()
+    .replace(/[^A-Z]/g, '') // remove all from str except A to Z alphabets
+
+  const adjacentDiffList = []
+
+  for (let i = 0; i < upperChars.length - 1; i++) {
+    // destructuring current char & adjacent char by index, cause in javascript String is an object.
+    const { [i]: char, [i + 1]: adjacentChar } = upperChars
+
+    if (char !== adjacentChar) {
+      adjacentDiffList.push(
+        Math.abs(char.charCodeAt() - adjacentChar.charCodeAt())
```

Comment:
I don't believe they used the abs in the specification.

Reply:
From the problems example, the increasing of "update" = `[5,12,3,19,15]` is **abs**, cause except **abs** the increasing should be `[ 5, 12, 3, -19, 15 ]`
difference between `a:97 - t:116 = -19 = |19|`

this is why I added the **abs** method.

Reply:
You are doing the opposite. Do `t:116 - a:97 = 19`. Otherwise everything will be considered an increasing.

Reply:
If I do `t:116 - a:97 = 19` this output should be the same. I've added more **not-increasing** words please check.